src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/GoalResourceMapperTest.kt

```diff
@@ -96,8 +96,10 @@ internal class GoalResourceMapperTest {
       steps = listOf(expectedStepResponse),
       createdAt = expectedDateTime,
       createdBy = "a.user.id",
+      createdByDisplayName = "",
       updatedAt = expectedDateTime,
       updatedBy = "another.user.id",
+      updatedByDisplayName = "",
```

Comment:
No need for TODO statements here as the test will break when the mapper is updated 👍